lib/sql.js

```diff
@@ -690,9 +690,23 @@ SQLConnector.prototype.update = function(model, where, data, options, cb) {
  */
 SQLConnector.prototype._replace = function(model, where, data, options, cb) {
   var stmt = this.buildReplace(model, where, data, options);
-  this.execute(stmt.sql, stmt.params, options, cb);
+  this.execute(stmt.sql, stmt.params, options, function(err, info) {
+    if (info.affectedRows === 0 || err) {
+      err = errorIdNotFoundForReplace(where.id);
+      cb(err);
+    } else {
+      cb();
+    }
+  });
 };
 
+function errorIdNotFoundForReplace(idValue) {
+  var msg = 'Could not replace. Object with id ' + idValue + ' does not exist!';
```

Comment:
I think we use "Model" instead of "Object" in other messages.

Also the message should be localized via `g.f`, right?